doc/manuals/devel/HttpHeaders.md

```diff
@@ -0,0 +1,354 @@
+List of HTTP Headers used by Orion
+----------------------------------------------------------------------------------------------------
+
+1)  Accept                  
```

Comment:
It would be great to use this ToC with links to the corresponding sections. You have an example here: doc/manuals/user/walkthrough_apiv2.md